### PR TITLE
squad/frontend/views.py: Add support for filtering project list by age

### PR DESCRIPTION
**Pull request by katieworton**
Add a group parameter for filtering projects in a group home their age as an alternative to limiting by count.

### CODE REVIEW

**Comment by katieworton** [comment]
Drafted code - need to add unit testing.

File: squad/frontend/views.py [hunk 1]
```diff
@@ -99,8 +100,19 @@ def group_home(request, group_slug):
     elif order_by == 'last_updated':
         projects_queryset = projects_queryset.order_by('-datetime')
 
-    display_all_projects = request.GET.get('all_projects') is not None
     num_projects = group.get_setting('DEFAULT_PROJECT_COUNT')
+
+    max_project_age_days = group.get_setting('MAX_PROJECT_AGE_DAYS')
```
Comment (chaws) [inline squad/frontend/views.py:105]: I thought it over, and what do you think about the name `SHOW_PROJECTS_ACTIVE_N_DAYS_AGO`?

**Comment by chaws** [comment]
Just one comment about the variable name, but LGTM!

**Comment by katieworton** [comment]
@chaws I have added unit tests and refactored things a bit (instead of passing `request` to the function, I pass in `user`, `order_by` and `display_all_projects` - makes a lot more sense to do it like this & made it easier to write the unit tests).

Let me know what you think!

**Review by chaws — APPROVED** [review]
LGTM! I think your tests covered all possible scenarios for this feature

**Pull request merged**